butterknife-compiler/src/main/java/butterknife/compiler/BindingSet.java

```diff
@@ -542,6 +551,16 @@ private boolean hasMethodBindings() {
     return false;
   }
 
+  private boolean hasOnTouchMethodBindings() {
+    for (ViewBinding bindings : viewBindings) {
+      if (bindings.getMethodBindings()
+          .containsKey(OnTouch.class.getAnnotation(ListenerClass.class))) {
```

Comment:
I'm open to suggestions if you are not happy with the explicit dependency on the OnTouch annotation